lib/github/ldap.rb

```diff
@@ -69,6 +73,10 @@ class Ldap
     def initialize(options = {})
       @uid = options[:uid] || "sAMAccountName"
 
+      # Keep a reference to these as default auth for a Global Catalog if needed
+      @admin_user = options[:admin_user]
+      @admin_password = options[:admin_password]
```

Comment:
Not a blocker but it might be worth passing these into `configure_user_search_strategy` (by passing through the `options` hash) to let that method decide if it needs these, so we don't have Global Catalog concerns leaking out unnecessarily.